content/posts/2021/release-0.8.0.md

```diff
@@ -1,53 +1,315 @@
 
-<!-- TODO: add sections:  -->
+The Common Workflow Language engine was upgraded from `cwltool`
+version 1 to version 3 and the compliance with the CWL reference test
+suite has been improved. This allows to use many new CWL constructs in
+your workflows.
 
-### Upgrade your cluster to 0.8.0
+### CPU and Disk quota accounting
+
+Optionally, the REANA cluster may have CPU and Disk quota accounting
+feature turned on. The new command `reana-client quota-show` will
+allow to see your quota limits and current usage:
+
+```console
+$ reana-client quota-show --resource cpu --report limit
+No limit.
+$ reana-client quota-show --resource disk --report limit
+No limit.
+```
+
+If the quota accounting feature is turned on, the `quota-show` command will also allow to display current usage:
+
+```console
+$ reana-client quota-show  --resource cpu --report usage -h
+36s
+
+$ reana-client quota-show  --resource disk --report usage -h
+2.61 GiB
+```
+
+If the feature is enabled, the REANA web interface will indicate disk
+usage in your workflow list:
+
+![quota-in-workflow-list](/images/reana-0.8.0-quota-workflow.png)
+
+The overall quota consumption can be tracked in your profile page:
+
+![quota-in-profile](/images/reana-0.8.0-quota-profile.png)
+
+### Improved workflow validation
+
+The `reana-client validate` command has been improved to include a
+possibility to validate the workflow against the remote REANA cluster
+you would like to use. The new option is named
+`--server-capabilities`:
+
+```console
+$ cd my-analysis
+$ reana-client validate --server-capabilities
+```
+
+The option will cover the scenarios where your workflow definition can
+be theoretically correct, but the remote REANA cloud does not support
+the desired feature. One example is the optional `compute_backend`
+declaration. If you would like to set`compute_backend` to
+`htcondorcern` in order to instruct the REANA cluster to use the
+HTCondor high-throughput compute backend for some job, but the remote
+REANA cluster where you are connected to does not support it, you
+would be warned about the mismatch.
+
+### Improved list command filtering
+
+The `reana-client list` command's performance has been optimised. If
+you have a large amount of workflows, the `list` will execute much
+faster.
+
+The default output focuses on tracking the status of the workflows. If
+you would like to track the workflow progress or the workspace disk
+usage, please use new options `--include-progress` and
+`--include-workspace-size`.
+
+The `list` command also adds a possibility to filter workflows by name
+or by status via the new `--filter` option. This can be convenient if
+you work on several projects ("project-a", "project-b", ...) and have
+run many workflows such as "project-a.1", "project-a.2", up to
+"project-z.1". If you are interested in listing the status of
+workflow runs from the "project-x" group only, you can use:
+
+```console
+$ reana-clent list --filter name=project-x
```

Comment:
Not sure if [`project-x`](https://www.imdb.com/title/tt1636826/) is the best example..

Reply:
Hehe, I didn't know about the reference. Renamed to `project-p` with the hope of evoking the generic naming habits like "Project P", "Workflow W", "Data D", etc. Hope it reads better. 

(FWIW, I had `myanalysis-a`, `myanalysis-b`, etc before, but it did not express the notion of completely separate "projects" that well, so swtiched to "project-a". But perhaps there is something even better.)